src/ui/home/HomeCtrl.ts

```diff
@@ -255,13 +258,13 @@ function fixSeeks(seeks: CorrespondenceSeek[]): CorrespondenceSeek[] {
   })
   return [
     ...seeks
```

Comment:
I'll setup prettier in the project to avoid these formatting changes in the future

Reply:
Yikes sorry about that! I didn't even realize it reformatted everything.